src/System.Runtime.Serialization.Formatters/src/Resources/Strings.resx

```diff
@@ -129,4 +129,223 @@
+    <value>Cannot supply both a MemberInfo and an Array to indicate the parent of a value type.</value>
+  </data>
+  <data name="Serialization_CorruptedStream" xml:space="preserve">
+    <value>Invalid BinaryFormatter stream.</value>
+  </data>
+  <data name="Serialization_HeaderReflection" xml:space="preserve">
+    <value>Header reflection error: number of value members: {0}.</value>
+  </data>
+  <data name="ArgumentNull_WithParamName" xml:space="preserve">
+    <value>Parameter '{0}' cannot be null.</value>
+  </data>
+  <data name="Serialization_Stream" xml:space="preserve">
+    <value>Attempting to deserialize an empty stream.</value>
+  </data>
+  <data name="Serialization_BinaryHeader" xml:space="preserve">
+    <value>Binary stream '{0}' does not contain a valid BinaryHeader. Possible causes are invalid stream or object version change between serialization and deserialization.</value>
+  </data>
+  <data name="Serialization_TypeExpected" xml:space="preserve">
+    <value>Invalid expected type.</value>
+  </data>
+  <data name="Serialization_StreamEnd" xml:space="preserve">
+    <value>End of Stream encountered before parsing was completed.</value>
+  </data>
+  <data name="Serialization_CrossAppDomainError" xml:space="preserve">
+    <value>Cross-AppDomain BinaryFormatter error; expected '{0}' but received '{1}'.</value>
+  </data>
+  <data name="Serialization_Map" xml:space="preserve">
+    <value>No map for object '{0}'.</value>
+  </data>
+  <data name="Serialization_Assembly" xml:space="preserve">
+    <value>No assembly information is available for object on the wire, '{0}'.</value>
+  </data>
+  <data name="Serialization_ObjectTypeEnum" xml:space="preserve">
+    <value>Invalid ObjectTypeEnum {0}.</value>
+  </data>
+  <data name="Serialization_AssemblyId" xml:space="preserve">
+    <value>No assembly ID for object type '{0}'.</value>
+  </data>
+  <data name="Serialization_ObjectTypeEnum" xml:space="preserve">
+    <value>Invalid ObjectTypeEnum {0}.</value>
+  </data>
+  <data name="Serialization_ArrayType" xml:space="preserve">
+    <value>Invalid array type '{0}'.</value>
+  </data>
+  <data name="Serialization_TypeCode" xml:space="preserve">
+    <value>Invalid type code in stream '{0}'.</value>
+  </data>
+  <data name="Serialization_TypeWrite" xml:space="preserve">
+    <value>Invalid write type request '{0}'.</value>
+  </data>
+  <data name="Serialization_TypeRead" xml:space="preserve">
+    <value>Invalid read type request '{0}'.</value>
+  </data>
+  <data name="Serialization_AssemblyNotFound" xml:space="preserve">
+    <value>Unable to find assembly '{0}'.</value>
+  </data>
+  <data name="Serialization_InvalidFormat" xml:space="preserve">
+    <value>The input stream is not a valid binary format. The starting contents (in bytes) are: {0} ...</value>
+  </data>
+  <data name="ArgumentNull_Stream" xml:space="preserve">
+    <value>Stream cannot be null.</value>
```

Comment:
Nit: This resource string could also probably be deleted. The one place it is used the param is named "stream" so this doesn't really add much more value to the default `ArgumentNullException` message, which includes the param name.